setup.py

```diff
@@ -67,6 +68,12 @@
             "spalloc-where-is = spalloc.scripts.where_is:main",
         ],
     },
+    # Booting directly needs rig; not recommended! Use SpiNNMan instead, as
+    # that has an up-to-date boot image pre-built
+    extras_require={
+        'boot': [
+            'rig',
```

Comment:
This looks kike an add of a rig requirement not a removal!

Reply:
It's describing what was there already; we were already (in one of the command line tools) looking to see if it was there and enabling extra options if it was.